proto/internal/temporal/server/api/adminservice/v1/request_response.proto

```diff
@@ -209,15 +209,15 @@ message AddSearchAttributesResponse {
 message RemoveSearchAttributesRequest {
     repeated string search_attributes = 1;
     string index_name = 2;
-    string namespace = 4;
+    string namespace = 3;
 }
 
 message RemoveSearchAttributesResponse {
 }
 
 message GetSearchAttributesRequest {
     string index_name = 1;
-    string namespace = 4;
+    string namespace = 2;
```

Comment:
This is just a fix in the arguments numbers. These are new arguments, and were not updated in tctl or cli yet. It should be safe.